app/assets/stylesheets/content/ui_select.sass

```diff
@@ -1,6 +1,7 @@
 // keep this rule if you want ui-select drag-n-drop to work properly
-.ui-select-match .ui-select-match-item *
-  pointer-events: none
+.ui-select-match .ui-select-match-item
+  span, span *
+    pointer-events: none
```

Comment:
and the child * selector is there to also block the pointer event for all the children of the span, because this rule behaves weirdly on the children